pandas/_libs/lib.pyx

```diff
@@ -246,7 +263,7 @@ def item_from_zerodim(val: object) -> object:
 
 @cython.wraparound(False)
 @cython.boundscheck(False)
-def fast_unique_multiple(list arrays, sort: bool=True):
```

Comment:
for functions that are neither cdef nor cpdef, we can use python-style annotations.

the vague/vain hope is that we can get this code close enough to being valid python that we can use python linters on it